src/Chapter3.hs

```diff
@@ -460,6 +491,8 @@ Create a simple enumeration for the meal types (e.g. breakfast). The one who
 comes up with the most number of names wins the challenge. Use your creativity!
 -}
 
+data MealTypes = Breakfast | Brunch | Lunch | TeaTime | Dinner | Snack | AfterPartyMeal
```

Comment:
Nice menu! Though, the party probably wasn't that great, if you still need a meal after that party :unamused: :rofl: